api/lib/infrastructure/serializers/jsonapi/session-serializer.js

```diff
@@ -85,6 +86,11 @@ module.exports = {
         'status',
         'examinerGlobalComment',
       ],
+      transform(session) {
+        const transformedSession = Object.assign({}, session);
+        transformedSession.status = session.computedStatus;
+        return transformedSession;
```

Comment:
Je ne suis pas 100% sure de ça , mais est-ce que ça ne reviendrait pas à faire : 
```javascript
return Object.assign({status: session.computedStatus}, session); 
```

Reply:
@Krysthalia Possible ;) . Be my guest

Reply:
Ça ne sera équivalent que si `session` n'a pas de propriété `status`.

```js
return Object.assign({}, session, {status: session.computedStatus}); 
```

Sinon, avec le _spread operator_ on peut généralement se passer de `Object.assign` :

```js
return { ...session, status: session.computedStatus };
```